File: python/ex20.py
L = float(input())
A = float(input())
C = float(input())
M = float(input())
latas = 0
area = L * A
latas = int(area // M)
if area % M != 0:
    latas +=1
custo = latas * C
print(latas)
print(f'{custo:.2f}')

File: c/ex20.c
//Pintando a parede
//Informado o custo da lata de tinta e quantos m² ela é capaz de pintar
//E as dimensões da parede, calcula-se o custo para pintar essa parede
#include<stdio.h>
int main(){
    float c, m, l, a;
    scanf(" %f %f %f %f", &l, &a, &c, &m);
    float area = l * a;
    int latas = area / m;
    if(latas * m != area){
        latas++;
    }
    printf("%i\n%.2f", latas, latas*c);
}